README.md

```diff
@@ -1 +1,50 @@
-# Module1-Capstone-Project
\ No newline at end of file
+# Module1-Capstone-Project
+This is a Capstone Project for Module 1 website showcasing the Global Sabre Summit. Someof the features are replicated from Behance. The project is to showcase the mastery of what I have learned In Microverse's module 1.
+## Built With
+* HTML
+* CSS
+* JavaScript
+* Git
+* Behance
+
+## Live Demo 
+
+[Live Demo](absolute link)
+
+## Getting Started
+
+To get a local copy up and running, follow these simple example steps.
+
+## Prerequisites
+* HTML, CSS, and JS
+* Git and Github
+* Terminal/cmd
+* VSCode or any other preferable IDE
+* 
+
+## Setup
+From the terminal, follow the following procedure:
+* Clone the repostory into your local machine
+use the command git clone
+* Navigate into the repostory with the command below
+cd module1 capstone project
+
+## Authors
+
+👤 **Collins Musoko**
+* Github: [@Carshy Collins](https://github.com/)
+* Linkedln: [@Collins Musoko](https://www.linkedin.com/feed/)
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Carshy/Hello-microverse/issues)
+## Show Support
+Give a ⭐️ if love this project 
+# Acknowledgements
```

Comment:
- [ ] As a professional, please add the designer name and a link to the original design